### PR TITLE
squid caching improvements

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: containers/squid.sh [hunk 1]
```diff
@@ -11,9 +11,18 @@ set -eu
...
 
-if [ "${1:-}" = start ]; then
+is_running() {
+    [ -n "$($CRUN ps -q -f 'name=^squid$')" ]
+}
+
+start() {
+    if is_running; then
+        echo "Already running"
+        return 0
```
Comment (jkonecny12) [inline containers/squid.sh:23]: Wouldn't it be better to use ``exit 0`` instead to avoid issues when the main part of this script is extended.

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

File: containers/squid.sh (hunk 1 above)
Comment (martinpitt) [reply containers/squid.sh:23]: No, I don't think so. The idea is that start() makes sure that the container is started, and that's the case if it's already running. If there is some subsequent action, then that should run no matter whether the container actually got started IMHO.

**Pull request merged**